client/src/components/payment/payment-summary.vue

```diff
@@ -30,11 +30,12 @@ export default class PaymentSummary extends Mixins(PaymentMixin, ReceiptMixin) {
   private loading = false
 
   private get receiptNumber (): string {
-    return `Receipt No. ${this.summary?.receipt.receiptNumber}`
+    const receiptNumber = this.summary?.receipt.receiptNumber || 'UNK'
+    return `Receipt No. ${receiptNumber}`
```

Comment:
Sentry reported random errors de-referencing an undefined receipt... This is a work-around instead of a fix. Hopefully we will find out soon why there are duplicate or undefined objects and fix it.